### PR TITLE
README : Add notes and examples regarding image sharing

### PR DESCRIPTION
**Pull request by bd-arc**
# Overview

I'm following up on @MateusAndrade's [comment](https://github.com/react-native-community/react-native-share/pull/535#issuecomment-513567797) in #535.

### CODE REVIEW

**Comment by MateusAndrade** [comment]
Thanks for helping us @bd-arc ! I will give a look at this later.

Also, i see that you tested your approach to Share imagens using RNFetchBlob on IOS only. Is there any problem related to using this on Android?

**Comment by bd-arc** [comment]
@MateusAndrade I'm pretty sure that this will work as is on both platforms, except that Android's specific permissions will need to be taken care of.

In this regard, it might be useful to link to what @xliank did [here](https://github.com/react-native-community/react-native-share/issues/533#issuecomment-508459727).

**Comment by MateusAndrade** [comment]
HI @bd-arc , sorry for the late answer. You update your branch with the last changes on README.MD?

File: README.md [hunk 1]
```diff
@@ -267,7 +269,113 @@ Share.shareSingle(shareOptions);
...
+
+* The previous examples have been tested on **iOS only** at that point.
+
+* When trying to share an image via Facebook, and if the Facebook app isn't installed, the plugin will default on opening the browser. **With a base64 image, it will actually crash the Facebook website**. Therefore, when using `shareSingle()`, it is recommended to check whether the app is installed or not thanks to a plugin like [this one](https://github.com/redpandatronicsuk/react-native-check-app-install).
+
+### Prerequisites to sharing images on iOS
+
+There are two requirements to make this work:
+
+1. **Your app's `Info.plist` file must contain an `NSPhotoLibraryUsageDescription` key**, otherwise it will crash as soon as you try to share your image on a real device — not on the simulator. Whether you're downloading the image first, importing a local image64 file or reading a remote URL, you need to include this permission.
```
Comment (alexwasner) [inline README.md:361]: You also need (or maybe only need?) `NSPhotoLibraryAddUsageDescription`

**Pull request closed without merging**